keras/utils/timeseries_dataset.py

```diff
@@ -73,6 +71,10 @@ def timeseries_dataset_from_array(
     end_index: Optional int; data points later (exclusive) than `end_index`
       will not be used in the output sequences.
       This is useful to reserve part of the data for test or validation.
+    seq_to_seq: bool; whether the target
```

Comment:
It is not very clear to me what's the usage for this flag.

Reply:
It's like many-to-many scenario but without zip it manually.